cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/sla/DynamicVmCreationByCpuUtilizationAndFreePesOfVm.java

```diff
@@ -184,15 +192,15 @@ private void printVmsCpuUsage(EventInfo eventInfo) {
      * time.
      *
      * @param cloudlet the Cloudlet to select a VM to
-     * @return the selected Vm
+     * @return the selected VM
      */
     private Vm selectVmForCloudlet(Cloudlet cloudlet) {
         List<Vm> createdVms = cloudlet.getBroker().getVmsCreatedList();
         System.out.println("\t\tCreated VMs: " + createdVms);
         Comparator<Vm> sortByNumberOfFreePes
                 = Comparator.comparingInt(vm -> getExpectedNumberOfFreeVmPes(vm, false));
         Comparator<Vm> sortByExpectedCloudletResponseTime
-                = Comparator.comparingDouble(vm -> getExpectedCloudletResponseTime(cloudlet, vm));
```

Comment:
If this method is sorting VMs by the expected response time of a given cloudlet, the VM must be included into the getExpectedCloudletResponseTime() method as it was before. Thus, this method must have a Vm parameter. See the next comment.